src/NServiceBus.Core/Serializers/Json/JsonMessageSerializer.cs

```diff
@@ -12,12 +12,11 @@ namespace NServiceBus.Serializers.Json
     using Newtonsoft.Json.Converters;
     using NServiceBus.Serialization;
 
-    /// <summary>
-    /// JSON message serializer.
-    /// </summary>
-    public class JsonMessageSerializer : IMessageSerializer
+    class JsonMessageSerializer : IMessageSerializer
```

Comment:
Both ways are good for me. Referencing json makes us internalize it. I'd probably go with handcrafting.